admin/assets/javascripts/custom-file-upload.js

```diff
@@ -25,8 +26,12 @@ $(function () {
     })
 
     $formElement.submit(function () {
+      if (isSubmitted) {
+        return false
+      }
       $formElement.val('Sending...')
-      $(this).attr('disabled', 'disabled')
```

Comment:
@jon-shipley wondering why this was removed?

Reply:
Hi @pris54 

Good question :)

This line was identified as causing IE8-11 to fail to upload the csv file.  It would supply a multi-part boundary but no actual content.  This was checked in detail  (I installed a proxy). My hypothesis is that on a fast machine IE would start the upload, the control would be disabled, and IE would *notice* and fail to send the file.

I assumed that this line was in there to prevent double-clicking submitting the form twice, and came up with an alternative using a boolean flag.

Reply:
@jon-shipley I knew you would have a good reason to remove it =) And yes, it's sole purpose was to disable the submit button after the first submission.